src/config/vim.zig

```diff
@@ -24,6 +24,21 @@ pub const ftplugin =
     \\
     \\let b:undo_ftplugin = 'setl cms< isk< ofu<'
     \\
+    \\if !exists('current_compiler')
+    \\  compiler ghostty
+    \\  let b:undo_ftplugin .= "| compiler make"
```

Comment:
I think we can just reset makeprg and errorformat here

```suggestion
    \\  let b:undo_ftplugin .= ' makeprg< efm<'
```

Reply:
done!